install/install.bat

```diff
@@ -1,6 +1,6 @@
 @echo off
 
-:: Script to install the blender nif scripts
+:: Script to install the Blender nif scripts
```

Comment:
Bit selfish, but can you remove any update to the install folder as it will cause me to rebase with conflicts :+1: 
Check out my PR as I did some refactoring on this area. 
I can include any of these changes in that PR no problem, just let me know..

Reply:
Will Do!

Reply:
Awesome